frontend/app/component/form/recounted/RecountedForm.test.tsx

```diff
@@ -1,120 +1,207 @@
+      await user.click(yes);
 
       const submitButton = screen.getByRole("button", { name: "Volgende" });
       await user.click(submitButton);
 
-      expect(spy).toHaveBeenCalledWith("http://testhost/api/polling_stations/1/data_entries/1", {
-        method: "POST",
-        body: JSON.stringify(expectedRequest),
-        headers: {
-          "Content-Type": "application/json",
-        },
-      });
+      expect(spy).toHaveBeenCalled();
+      const { url, method, body } = getUrlMethodAndBody(spy.mock.calls);
+
+      expect(url).toEqual("http://testhost/api/polling_stations/1/data_entries/1");
+      expect(method).toEqual("POST");
+      expect(body).toEqual(expectedRequest);
 
       const result = await screen.findByTestId("result");
       expect(result).toHaveTextContent(/^Success$/);
     });
   });
 
-  test.skip("422 response results in display of error message", async () => {
+  test("422 response results in display of error message", async () => {
     overrideOnce("post", "/api/polling_stations/1/data_entries/1", 422, {
       message: "422 error from mock",
     });
 
     const user = userEvent.setup();
 
-    render(<RecountedForm />);
+    render(Component);
+
+    const no = screen.getByTestId("no");
+    await user.click(no);
 
     const submitButton = screen.getByRole("button", { name: "Volgende" });
     await user.click(submitButton);
-    const result = await screen.findByTestId("result");
-    expect(result).toHaveTextContent(/^Error 422 error from mock$/);
+    const feedbackServerError = await screen.findByTestId("feedback-server-error");
+    expect(feedbackServerError).toHaveTextContent(/^Error422 error from mock$/);
+
+    expect(screen.queryByTestId("result")).not.toBeNull();
+    expect(screen.queryByTestId("result")).toHaveTextContent(/^422 error from mock$/);
   });
 
-  test.skip("500 response results in display of error message", async () => {
+  test("500 response results in display of error message", async () => {
     overrideOnce("post", "/api/polling_stations/1/data_entries/1", 500, {
       message: "500 error from mock",
       errorCode: "500_ERROR",
     });
 
     const user = userEvent.setup();
 
-    render(<RecountedForm />);
+    render(Component);
+
+    const no = screen.getByTestId("no");
+    await user.click(no);
 
     const submitButton = screen.getByRole("button", { name: "Volgende" });
     await user.click(submitButton);
-    const result = await screen.findByTestId("result");
-    expect(result).toHaveTextContent(/^Error 500_ERROR 500 error from mock$/);
+    const feedbackServerError = await screen.findByTestId("feedback-server-error");
+    expect(feedbackServerError).toHaveTextContent(/^Error500 error from mock$/);
+
+    expect(screen.queryByTestId("result")).not.toBeNull();
+    expect(screen.queryByTestId("result")).toHaveTextContent(/^500 error from mock$/);
+  });
+
+  describe("RecountedForm errors", () => {
+    test("No radio selected", async () => {
```

Comment:
The other similar tests mention the code of the error in the name, so would be good to have the F.02 in there.

Reply:
Changed in b90d2a8d33a757b4a0ada6d31a3ade9b59aab4f8